nevergrad/benchmark/frozenexperiments.py

```diff
@@ -4,16 +4,92 @@
 # LICENSE file in the root directory of this source tree.
 
 import typing as tp
+from uuid import uuid4 as uuid_gen
 import numpy as np
 from nevergrad import optimizers
 from nevergrad.optimization.base import ConfiguredOptimizer
 from nevergrad.optimization import experimentalvariants  # pylint: disable=unused-import
 from nevergrad.functions import ArtificialFunction
+from nevergrad.functions.perfcap3d.core import Perfcap3DExperimentConfig, Perfcap3DFunction, Perfcap3DServerComm
```

Comment:
for simplicity it would be better to only do `from nevergrad.functions import perfcap3d` since all the casses are imported in the `__init__.py`

Also, are all these classes necessary? It seems too complex for me from a user perspective, wouldnt it work to have `Perfcap3DFunction` take an `experiment_filename` and then set up all the necessary classes itself?

Reply:
I added a simpler interface in this new version